multinet/auth/google.py

```diff
@@ -0,0 +1,159 @@
+"""Handling of Google Authorization."""
+import requests
+import base64
+import json
+
+from os import getenv
```

Comment:
Is there a specific reason to pluck `getenv` out like this? In general I prefer for stdlib functions to be fully qualified (i.e., change this to just `import os`, and change `getenv()` throughout to `os.getenv()`).

Also, what's the reason for the linebreaks between the blocks of imports?

Reply:
Valid, I'll change it. As for the spacing, I tend to separate straight imports (`import os`) from "plucked" imports (`from os import getenv`). No real reason besides that, just seems stylistically  right.